server/src/daemon/collection.rs

```diff
@@ -0,0 +1,206 @@
+// org.freedesktop.Secret.Collection
+
+use std::{
+    collections::HashMap,
+    sync::{atomic::AtomicBool, Arc, Mutex},
+    time::Duration,
+};
+
+use oo7::{
+    dbus::api::{Properties, SecretInner},
+    portal::{Item, Keyring},
+};
+use tokio::sync::RwLock;
+use zbus::{interface, zvariant, ObjectServer, SignalContext};
+use zvariant::{ObjectPath, OwnedObjectPath};
+
+use super::{
+    error::ServiceError, prompt::Prompt, secret::Secret, service_manager::ServiceManager, Result,
+    Service,
+};
+use crate::SECRET_COLLECTION_PREFIX;
```

Comment:
this is not a prefix, please name the variables according to their actual content

Reply:
I was following the gnome-keyring naming style: https://gitlab.gnome.org/GNOME/gnome-keyring/-/blob/master/daemon/dbus/gkd-secret-types.h#L38. This constant defines the "pre" part of an ObjectPath.